### DISCUSSION TITLE
无法添加模版到笔记，无法新建工作区笔记

### DISCUSSION THREAD

**Discussion by npj06** [windingwind/zotero-better-notes#991] [Q&A] [answered]
请问大家，插件工作逻辑是更改了吗？
现在没办法设置工作区笔记，但是模版新建只能到工作区笔记中，因此插入的模版也不知道到底去哪儿了
完全不知道怎么用了....
<img width="1365" alt="截屏2024-06-03 11 11 43" src="https://github.com/windingwind/zotero-better-notes/assets/114918453/4f643833-70d6-4b6a-b128-dc77b3f52201">
<img width="254" alt="截屏2024-06-03 11 12 10" src="https://github.com/windingwind/zotero-better-notes/assets/114918453/7078b6c0-176b-402c-82c8-cd480e50a945">
<img width="259" alt="截屏2024-06-03 11 12 27" src="https://github.com/windingwind/zotero-better-notes/assets/114918453/1f2e7ce9-3d33-4d5c-8e10-1ac8ed698d76">
<img width="797" alt="截屏2024-06-03 11 13 12" src="https://github.com/windingwind/zotero-better-notes/assets/114918453/214c37ae-433b-4a62-8a05-cfe6b77bef4d">
@windingwind at一下作者大大
zotero：7.0.0-beta.
插件版本；1.1.4-beta.77

**Comment by windingwind** [comment, marked as answer]
#995
